GCDWebServer/Core/GCDWebServerPrivate.h

```diff
@@ -92,6 +92,13 @@
 
 #define __GCDWEBSERVER_LOGGING_FACILITY_COCOALUMBERJACK__
 
+/**
+ * Whether async should be used by log messages, excluding error messages that are always sent sync.
+ **/
+#ifndef LOG_ASYNC_ENABLED
+    #define LOG_ASYNC_ENABLED YES
```

Comment:
OK, so it seems what we need is 2 commits in one PR or 2 PRs of 1 commit, it doesn't really matter:
- 1 to add the `.swift-version` file which is for CocoaPods
- 1 to bump the version of CocoaLumberJack and add `LOG_ASYNC_ENABLED` to it builds

Does that make sense?

(in my projects I always want to have 1 distinct commit per independent change)